x/authz/client/cli/tx.go

```diff
@@ -47,7 +47,6 @@ func GetTxCmd(ac address.Codec) *cobra.Command {
 
 	AuthorizationTxCmd.AddCommand(
 		NewCmdGrantAuthorization(ac),
-		NewCmdRevokeAuthorization(ac),
 		NewCmdExecAuthorization(),
```

Comment:
will the other commands be removed as well?

Reply:
they have custom logic in tx.go

Reply:
Let's add a comment for them as suggested by Marko: https://github.com/cosmos/cosmos-sdk/pull/17959#discussion_r1348435335